mbtiles/src/summary.rs

```diff
@@ -206,7 +208,7 @@ fn get_zoom_precision(zoom: u8) -> usize {
 }
 
 fn webmercator_to_wgs84(x: f64, y: f64) -> (f64, f64) {
-    let lng = (x / 6378137.0).to_degrees();
+    let lng = (x / 6_378_137.0).to_degrees();
```

Comment:
this number is `EARTH_CIRCUMFERENCE÷pi÷2` - i.e. `EARTH_RADIUS` - may want to create a const for that too.